lib/vdsm/network/link/setup.py

```diff
@@ -55,3 +74,15 @@ def parse_nets_bridge_opts(nets):
                 opt.split('=', 1) for opt in opts_str.split(' ')
             )
             yield name, bridge_opts
+
+
+def parse_nets_bridge_opts_for_nmstate(opts):
+    if opts:
+        opts_str = opts.replace('_', '-')
+        bridge_opts = dict(
+            opt.split('=', 1) for opt in opts_str.split(' ')
+        )
```

Comment:
This part is very similar to `_string_to_dict` above. Please resuse that function.